soh/soh/Enhancements/controls/GameControlEditor.cpp

```diff
@@ -216,36 +215,132 @@ namespace GameControlEditor {
         ImGui::EndTable();
     }
 
+    // Copied from InputEditor::DrawButton
```

Comment:
is there a way we could do this without copypasta?

Reply:
Initially when I tried to use LUS's code, I ran into the issue where the Input Editor window's current tab is stored in the Input Editor class and was what was being referenced in the Draw Button function.  If we want to remove the copypasta for a v2, it will thus require changes to the LUS Input Editor to make Draw Button (as well as the controller selection dropdown in your next comment) agnostic to what window the button is being drawn in, which I wanted to avoid at the moment due to all the refactoring going on with LUS, and the fact that it might need some more discussion in order to be aligned with LUS's game agnosticism.  

Personally, I would argue that any game _could_ have custom controls like SOH now does, and therefore having all of LUS's ImGui wrappers usable in a game-owned window would be great, since I know we already are doing that for things like checkboxes, so I do hope we can get rid of this copypasta at some point.  This whole custom controls window will need a lot of cleanup work in the future, and its current implementation was deemed acceptable since we wanted to just get it out there and have a place to put all the stuff that will be moved out of LUS in the near future.

Reply:
thanks for explaining. would you mind adding a couple todos and/or making an issue for removing the copypasta?

Reply:
I already have #1242 for some of the work needed on the Game Control Editor, so I'll specifically call out the copypasta there!